doc/sphinx/source/recipes/recipe_sea_surface_salinity.rst

```diff
@@ -202,8 +202,14 @@ References
 Example plots
 -------------
 
-.. figure:: /recipes/figures/sea_surface_salinity/radar.png
+.. figure:: /recipes/figures/sea_surface_salinity/radar_bias.png
    :align: center
 
-   Radar plot showing correlation of average sea surface salinity for multiple
+   Radar plot showing bias of average sea surface salinity for multiple
```

Comment:
change lines 208-209 to "Radar plot showing the mean state biases (simulation minus observations) for the regional averages of sea surface salinity in the selected ocean basins and seas"